01_version_control/my_favorite_neat_little_git_trick_demo.md

```diff
@@ -55,6 +58,7 @@
 
 What does it do?
 
+- When pushing new branches, automatically set the upstream branch to a branch with the same name on the primary remote
```

Comment:
This I don't understand. Isn't this done anyway?

```bash
git push -u origin new_branch
```

Or does this mean I don't have to use the `-u` flag?